.npmignore

```diff
@@ -0,0 +1,12 @@
+*.log
+node_modules
+src
+tests
+.github
+dangerfile.js
+.travis.yml
+.gitattributes
+.babelrc
+.eslintrc
+CODE_OF_CONDUCT.md
```

Comment:
we should add 
```
jestConfig.js
jestSetup.js
Dockerfile
terra.png
CONTRIBUTORS.md
```

Reply:
This has landed in master. https://github.com/cerner/terra-enzyme-intl/blob/master/.npmignore#L13-L17